pkg/controller/nodeipam/ipam/multinetwork_cloud_cidr_allocator_test.go

```diff
@@ -15,7 +15,12 @@ const (
 	gkeNetworkParamsKind = "GKENetworkParams"
 )
 
-func network(name, gkeNetworkParamsName string) *networkv1.Network {
+func network(name, gkeNetworkParamsName string, isReady bool) *networkv1.Network {
```

Comment:
Did you intend to add a unit test for getUpNetworks?

Reply:
I'm not sure what Maciej originally intended, but I have added a unit test for getUpNetworks.